docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openjdk:8-jre-slim
+FROM eclipse-temurin:8u372-b07-jre-jammy
```

Comment:
we can use this image on foxbox and make sure this works. We can test this on foxbox. 
Let me know I can be around.

So, I am thinking vanta won't flag this base for any severe issues.

Reply:
There are few medium security w/o fixed versions but the deadline is far away then critical security.